main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"crypto/tls"
+	"log"
+	"os"
+
+	"github.com/cloudfoundry/noaa/consumer"
+	"github.com/wavefronthq/cloud-foundry-nozzle-go/nozzle"
+)
+
+var logger = log.New(os.Stdout, "[WAVEFRONT] ", 0)
+var debug = os.Getenv("WAVEFRONT_DEBUG") == "true"
+
+func main() {
+	// if debug {
```

Comment:
Uncomment these lines?

Reply:
I think is best if we log always the env, just few lines that will allow easy support without restarting.